<launch>

  <node output="screen" name="rosapi" pkg="rosapi" type="rosapi.py" />
  <node output="screen" name="rosbridge" pkg="rosbridge_server" type="rosbridge.py" />
  
  <node name="im_tunnel" pkg="demo_interactive_markers" type="im_tunnel">
    <remap from="target_frame" to="/base_link"/>
    <remap from="topic_ns" to="pr2_marker_control"/>
  </node>
  
</launch>
